synapse/cortex.py

```diff
@@ -1453,6 +1453,9 @@ def _initCorePerms(self):
             {'perm': ('storm', 'macro', 'edit'), 'gate': 'cortex',
              'desc': 'Controls access to edit a storm macro.'},
 
+            {'perm': ('storm', 'lib', 'stix', 'export', 'maxsize'), 'gate': 'cortex',
```

Comment:
This should be defined on the `LibStixExport` class in the `_storm_lib_perms` array.

Reply:
Thanks! I totally forgot about that :+1: